vkwave-bots-storage/vkwave_bots_storage/storages/redis.py

```diff
@@ -0,0 +1,109 @@
+import asyncio
+import json
+import typing
+
+import aioredis
+from vkwave_bots_storage.base import AbstractExpiredStorage, NO_KEY, NoKeyOrValue
+
+Key = typing.NewType("Key", str)
+Value = typing.Any
+TTL = typing.NewType("TTL", float)
+
+Dumper = typing.Callable[[Value], str]
+Loader = typing.Callable[[str], Value]
+
+
+class RedisStorage(AbstractExpiredStorage):
+    def __init__(
+            self,
+            host: str = 'localhost',
+            port: int = 6379,
+            db: typing.Optional[int] = None,
+            password: typing.Optional[str] = None,
+            ssl=None,
```

Comment:
We need types there. For loop argument it too. It should be `Optional[...] = None`